tests/theia_automation_lsp/inc/theia/lsp_constants.py

```diff
@@ -15,3 +15,14 @@
 TAB_TITLE_TEMPLATE = "{0}_{1}.cbl"
 
 SAVE_DIALOG_TEMPLATE = "Do you want to save the changes you made to {0}?"
+
+COPYBOOK_FOLDER = ".copybooks"
+ZOWE_PROFILE = "zowe-profile-1"
+
+CPY_DIR_1 = "DATA.SET.PATH1"
+CPY_DIR_2 = "DATA.SET.PATH2"
+
+USER_COPYBOOK_PREFERENCES = {
+    "broadcom-cobol-lsp.cpy-manager.paths": [CPY_DIR_1, CPY_DIR_2],
+    "broadcom-cobol-lsp.cpy-manager.profiles": ZOWE_PROFILE
```

Comment:
I think is necessary avoid this hardcoded keys and introduce a constant definition, because sooner will change and we need an easy way to update this key

Reply:
I think that these names could be hardcoded. Because they are related to our file structure for testing. 
`ZOWE_PROFILE` and `CPY_DIR_1` are part of the folder path:
https://github.com/grianbrcom/che-che4z-lsp-for-cobol/tree/moreTestCases/tests/theia_automation_lsp/test_files/cobol/.copybooks/zowe-profile-1/DATA.SET.PATH1
And in tests, we must use `tests/theia_automation_lsp/test_files/cobol` as our project folder.

Reply:
Sorry I wasn't clear: I am referring to the keys "broadcom-cobol-lsp.cpy-manager.paths", "broadcom-cobol-lsp.cpy-manager.profiles" and so on.
Those keys IMO should ke kept (if possible) in one place so that we can easily change it.
Just to make an example: actually I have a PR to introduce a new key "broadcom-cobol-lsp.cpy-manager.paths.local" and rename the zowe ones in "broadcom-cobol-lsp.cpy-manager.paths.dsn".
Now if the old keys are defined in one place I am ok with that, if in the code base more than one time you have something like 
```JSON
{
   "broadcom-cobol-lsp.cpy-manager.paths": [CPY_DIR_1, CPY_DIR_2],
    ...
}
```
it could be a problem from maintainability prospective ^__^

Reply:
Oh, you are right!